reconcile/change_owners/change_owners.py

```diff
@@ -231,6 +231,20 @@ def init_gitlab(gitlab_project_id: str) -> GitLabApi:
     return GitLabApi(instance, project_id=gitlab_project_id, settings=settings)
 
 
+def assert_restrictive(changes: list[BundleFileChange], user: str):
+    for change in changes:
+        for dc in change.diff_coverage:
+            for c in dc.coverage:
+                if c.change_type_processor.restrictive:
+                    approvers = {a.org_username for a in c.approvers}
+                    if user not in approvers:
+                        logging.error(
+                            f"change type {c.change_type_processor.name} is restrictive"
+                            f"user  {user} is not an approver"
+                        )
+                        sys.exit(1)
```

Comment:
should we allow anyone to submit, but need a `/good-to-test` handler of some sort?

Reply:
Yeah, I'd add that in a new iteration